bucket/i2p-portable.json

```diff
@@ -0,0 +1,67 @@
+{
+    "version": "0.9.41",
+    "description": "An anonymizing network.",
+    "homepage": "https://geti2p.net",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://github.com/i2p/i2p.i2p/blob/master/LICENSE.txt"
+    },
+    "suggest": {
+        "JDK": [
+            "java/oraclejdk",
+            "java/openjdk"
+        ]
+    },
+    "url": "https://launchpad.net/i2p/trunk/0.9.41/%2Bdownload/i2pinstall_0.9.41_windows.exe#/dl.7z",
+    "hash": "e8d896e94696c3cb3be20cb28ea902595f82f7028622931a8ab6f451e4a3d6b5",
+    "bin": [
+        "I2P.exe",
+        "I2Psvc.exe"
+    ],
+    "shortcuts": [
+        [
```

Comment:
Then why do yo uadd this shortcut when it breaks portable mode. I would expect from `-portable` manifest to always use portable mode.